tasks/vesting.ts

```diff
@@ -688,8 +686,9 @@ const GET_BLOCKCHAIN_SUBTASK = {
   ): Promise<ParsedGrants> => {
     const totalSupply = await lNori.totalSupply();
     hre.log(`Total supply: ${totalSupply}`);
-    const rawBlockchainGrants: LockedNORI.TokenGrantDetailStructOutput[] =
```

Comment:
I was running into an issue with this line after cleaning and building locally after my changes that was upset that `LockedNORI` is only a type but was being used as a namespace here... so I deleted the type for now but maybe there's a way around this that I couldn't find. (tried adjusting the import and it just autocorrects to `import type {...`